src/testUtils.ts

```diff
@@ -78,29 +82,49 @@ export function getTestFlags(goConfig: vscode.WorkspaceConfiguration, args: any)
  * @param the URI of a Go source file.
  * @return test function symbols for the source file.
  */
-export function getTestFunctions(doc: vscode.TextDocument): Thenable<vscode.SymbolInformation[]> {
+export function getTestFunctions(doc: vscode.TextDocument, checker: prefixChecker): Thenable<vscode.SymbolInformation[]> {
 	let documentSymbolProvider = new GoDocumentSymbolProvider();
 	return documentSymbolProvider
 		.provideDocumentSymbols(doc, null)
 		.then(symbols =>
 			symbols.filter(sym =>
 				sym.kind === vscode.SymbolKind.Function
-				&& hasTestFunctionPrefix(sym.name))
+				&& checker(sym.name))
 		);
 }
 
 /**
+ * Function type for function that given a function name has
+ * returns whether it is of a certain type of prefix.
+ *
+ * @param the function name.
+ * @return whether the name has a function prefix.
+ */
+type prefixChecker = (name: string) => boolean;
+
+/**
  * Returns whether a given function name has a test prefix.
  * Test functions have "Test" or "Example" as a prefix.
  *
  * @param the function name.
  * @return whether the name has a test function prefix.
  */
-function hasTestFunctionPrefix(name: string): boolean {
+export function hasTestFunctionPrefix(name: string): boolean {
```

Comment:
There was no real point of making this a separate function in the first place (my apologies).

Instead of exporting this prefix function and adding another for benchmark, we can simplify things as below
- remove `hasTestFunctionPrefix` and move the prefix check into `getTestFunctions`
- create and export a new function `getBenchmarkFunctions` which will have the same code as `getTestFunctions` but with a different prefix check.

It will definitely get simpler when importing these functions in other files